src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,172 @@ $pf-chart-color-black-200: $pf-color-black-400;
+$pf-chart-candelstick--candleColors--positive: #fff; // grayscale theme
+$pf-chart-candelstick--candleColors--negative: "charcoal"; // grayscale theme
+
+// Chart
+$pf-chart-chart--BorderWidth: 1 solid;
+
+// Donut Chart
+$pf-chart-donut--color-threshold--first: $pf-color-black-200;
+$pf-chart-donut--color-threshold--second: $pf-color-black-300;
+$pf-chart-donut--color-threshold--third: $pf-color-black-400;
+$pf-chart-donut--color-threshold--success: $pf-chart-color-blue-300;
+$pf-chart-donut--color-threshold--warning: $pf-chart-color-gold-400;
+$pf-chart-donut--color-threshold--danger: $pf-color-red-100;
+$pf-chart-donut--label-subtitle--color: $pf-color-black-500;
+
+// Error Bar
+$pf-chart-errorbar--BorderWidth: 8;
+$pf-chart-errorbar--data--BackgroundColor: transparent;
+$pf-chart-errorbar--data--Opacity: 1; // material theme
+$pf-chart-errorbar--data-stroke--Width: 2;
+$pf-chart-errorbar--data-stroke--Color: "charcoal"; // grayscale theme
+
+// Group
+$pf-chart-group--colorScale: "colors"; // grayscale theme
+
+// Legend
+$pf-chart-legend--gutter--Width: 20; // this is 10px in the grayscale theme
+$pf-chart-legend--orientation: "horizontal"; // this is vertical in material
+$pf-chart-legend--title--orientation: "top";
+$pf-chart-legend--data--type: "square"; // this is circle in the grayscale theme
+$pf-chart-legend--title--Padding: 2; // this is 5px in grayscale theme
+$pf-chart-legend--colorScale: "colors"; // grayscale theme
+
+// Line Chart
+$pf-chart-line--data--BackgroundColor: transparent;
+$pf-chart-line--data--Opacity: 1; // material theme
+$pf-chart-line--data--stroke--Width: 2;
+$pf-chart-line--data--stroke--Color: "charcoal"; // grayscale theme
+
+// Pie Chart
+$pf-chart-pie--Padding: 8; // 50 in grayscale theme
+$pf-chart-pie--data--Padding: 8; // this is 10px in grayscale theme
+$pf-chart-pie--data--stroke--Width: 1;
+$pf-chart-pie--data--stroke--Color: transparent; // grayscale theme
+$pf-chart-pie--labels--Padding: 8; // this is 20 in grayscale theme
+$pf-chart-pie--static--Height: 230; // 400 in grayscale theme
+$pf-chart-pie--static--Width: 230; // 400 in grayscale theme
+$pf-chart-pie--dynamic--Height: 202;
+$pf-chart-pie--dynamic--Width: 202;
+$pf-chart-pie--padAngle: 1; // What does padAngle mean?
+$pf-chart-pie--colorScale: "colors";
+$pf-chart-pie--Height: 400; // grayscale theme --> whats the diff between dynamic/static and height
+$pf-chart-pie--Width: 400; // grayscale theme
+
+// Scatter Chart
+$pf-chart-scatter--data--stroke--Color: transparent;
+$pf-chart-scatter--data--stroke--Width: 0;
+$pf-chart-scatter--data--Opacity: 1; // material theme
+$pf-chart-scatter--data--BackgroundColor: "charcoal"; // grayscale
+
+// Scatter Chart
+$pf-chart-stack--data--stroke--Width: 1;
+$pf-chart-stack--colorScale: "colors";
+
+// Tooltip
+$pf-chart-tooltip--cornerRadius: 0; // this is 5 in grayscale theme
+$pf-chart-tooltip--pointerLength: 10; // grayscale theme
+$pf-chart-tooltip--flyoutStyle--cornerRadius: 0;
```

Comment:
can we just hyphenate names that aren't CSS properties? so this would be `corner-radius`. Also could we use `BorderRadius` here?

Reply:
Updated all names that aren't CSS properties to have hyphens.

@dlabrecq do you know if corner radius and border radius are the same things?

Reply:
corner radius is different than border. For example, we want the cornerRadius=0 so our  tooltip boxes look be square.